.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+## Jira Task
+- [MBU-?](https://memebattle.atlassian.net/browse/MBU-?)
+
+### Description
+<!--- Description about what have been done on this merge request, could include the objective of the jira ticket or some more technical detail of the merge requests (example: i had to do X thing on our redux store because of Y, i added a new utility function that makes Z, i choose to use X design pattern because of W) that will help who is reviewing to have a better understand about what you have done. -->
+
+### Commits Made
+- Commit 1: Briefly describe the changes made in the first commit.
+- Commit 2: Briefly describe the changes made in the second commit.
+
+### Checklist
+- [x] Updated the CHANGELOG.md with the latest changes.
```

Comment:
```suggestion
- [ ] Updated the CHANGELOG.md with the latest changes.
```

This checkbox should init empty, the idea is the devs check manually when they are creating the PR